src/igvfd/schemas/whole_organism.json

```diff
@@ -0,0 +1,62 @@
+{
+    "title": "Whole Organism",
```

Comment:
I think the title should be "Whole organism", not both words capitalized

Reply:
Done